assignment2/index.js

```diff
@@ -0,0 +1,103 @@
+var express = require("express");
+var mongoose = require("mongoose");
+var passwordHash = require("password-hash");
+var app = express();
+var bodyParser = require("body-parser");
+var async = require("async");
```

Comment:
`async` is an inbuilt module so no-need to import that

Reply:
sir without async its throwing error async is not define.